specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,530 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "v1.0-preview",
+      "title": "Cognitive Service Form Recognizer API V1 (Preview)",
+      "description": "Extracts information from forms and images into structured data based on a model created by a set of representative training forms."
+    },
+    "securityDefinitions": {
+      "apim_key": {
+          "type": "apiKey",
+          "name": "Ocp-Apim-Subscription-Key",
+          "in": "header"
+      }
+    },
+    "security": [
+      {
+          "apim_key": []
+      }
+    ],
```

Comment:
`x-ms-parameterized-host`?

Reply:
same as face.json as it too does not have it.